cpp/ql/src/JPL_C/LOC-4/Rule 20/PreprocessorUsePartial.ql

```diff
@@ -12,7 +12,7 @@
 import cpp
 
 predicate incomplete(Macro m) {
-  exists(string body | body = m.getBody() and not m.getBody().matches("%\\") |
+  exists(string body | body = m.getBody() and not m.getBody().matches("%\\\\") |
```

Comment:
Hmm, it looks like this bit is trying to exclude multi-line Macros from the query (I'm not 100% sure why).  The relationship between `matches` and `\` is fiddly though, and both affected queries lack explanation or tests - I think we will have to address this before we can be confident of a correct fix.